src/components/Widget/IntegrationsWidget.tsx

```diff
@@ -0,0 +1,477 @@
+            alt="webhooks"
+          />
+        ),
+      },
+    ],
+  },
+  {
+    title: CLOUD_VENDOR,
+    items: [
+      {
+        name: 'Amazon Web Services',
+        id: 'aws',
+        value: CLOUD_VENDOR,
+        icon: <ImageWithPlaceholder className="aws-logo" src="/apps/frontend-assets/partners-icons/aws.svg" alt="aws logo" />,
+      },
+      {
+        name: 'Google Cloud',
+        id: 'google_cloud',
+        value: CLOUD_VENDOR,
+        icon: (
+          <ImageWithPlaceholder
+            className="google-logo"
+            src="/apps/frontend-assets/partners-icons/google-cloud-short.svg"
+            alt="google logo"
+          />
+        ),
+      },
+      {
+        name: 'Microsoft Azure',
+        id: 'azure',
+        value: CLOUD_VENDOR,
+        icon: (
+          <ImageWithPlaceholder
+            className="azure-logo"
+            src="/apps/frontend-assets/partners-icons/microsoft-azure-short.svg"
+            alt="azure logo"
+          />
+        ),
+      },
+      {
+        name: 'Oracle Cloud Infrastructure',
+        id: 'oracle_cloud',
+        value: CLOUD_VENDOR,
+        icon: (
+          <ImageWithPlaceholder
+            className="oracle-logo"
+            src="/apps/frontend-assets/partners-icons/oracle-short.svg"
+            alt="oracle logo"
+          />
+        ),
+      },
+    ],
+  },
+  {
+    title: REDHAT_VENDOR,
+    items: [
+      {
+        name: 'OpenShift Container Platform',
+        id: 'openshift',
+        value: REDHAT_VENDOR,
+        icon: (
+          <ImageWithPlaceholder
+            className="redhat-icon"
+            src="/apps/frontend-assets/red-hat-logos/stacked.svg"
+            alt="red hat logo"
+          />
+        ),
+      },
+    ],
+  },
+];
+
+const IntegrationsWidget: FunctionComponent = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [expandedIndex, setExpandedIndex] = useState<number | null>(0);
+  const [integrationCounts, setIntegrationCounts] = useState<{ [key: string]: number }>({});
+
+  const [selectedTileValue, setSelectedTileValue] = useState<string>('');
+  const [isIntegrationsWizardOpen, setIsIntegrationsWizardOpen] = useState(false);
+  const [isSourcesWizardOpen, setIsSourcesWizardOpen] = useState(false);
+
+  const handleTileClick = (value: string) => {
+    setSelectedTileValue(value);
+        if ([REDHAT_VENDOR, CLOUD_VENDOR].includes(value)) {
+        setIsSourcesWizardOpen(true);
+      } else if ([COMMUNICATIONS, REPORTING, WEBHOOKS].includes(value)) {
+        setIsIntegrationsWizardOpen(true);
+      }
+  };
+
+  const allItems = integrationsData.flatMap((category) => category.items);
+  const sortedItems = allItems.sort((a, b) => a.name.localeCompare(b.name));
+
+  const onToggle = (index: number) => {
+    setExpandedIndex(expandedIndex === index ? null : index);
+  };
+
+  const fetchCloudSources = async () => {
```

Comment:
This file is pretty big. Could we move the api calls to a different file?

Reply:
Good point! Let's create a file for each of these requests so we are focused on one thing at the time.